cg-spec/editors_draft.html

```diff
@@ -2724,18 +2753,50 @@ <h2>Referential opacity</h2>
         #### but it does NOT entail
 
         << :linköping :population "104232"^^xsd::integer >>
-            source <https://en.wikipedia.org/wiki/Link%C3%B6ping> .
+            source <https://en.wikipedia.org/wiki/Linköping> .
         -->
         </pre>
 
-        <p>Note that the current semantics does not forbid the additional inferences to be drawn. However, they would require additional knowledge and an appropriate <a data-cite="RDF11-MT#dfn-semantic-extension">semantic extension</a>, not specified in this document. This makes RDF-star less readily usable for (but not incompatible with) that latter kind of use-cases.</p>
+
+      <p>Referentially opaque semantics of embedded triples has unfortunate effects on common data integration procedures. As the following example illustrates it puts annotated statements at risk of fragmentation.</p>
+
+      <pre id="ref-opacity-owl-entailment-2"
+           data-transform="updateExample"
+           data-content-type="text/turtle"
+           class="nohighlight example"
+      >
+        <!--
+        #### under OWL-entailment
+
+        :car owl:sameAs https://www.wikidata.org/wiki/Q1420.
+        << :alice :buys :car >> :documentedBy :ellen.
+
+        #### does NOT entail
+
+        << :alice :buys https://www.wikidata.org/wiki/Q1420 >> :documentedBy :ellen.
+
+        #### Therefore an annotated statement construct
+
+        :alice :buys :car.
+        << :alice :buys :car >> :documentedBy :ellen.
+
+        #### after a standard data integration procedure may fall apart
+
+        :alice :buys https://www.wikidata.org/wiki/Q1420.
+        << :alice :buys :car >> :documentedBy :ellen.
+        -->
+        </pre>
+
+
```

Comment:
```suggestion
```
This extra blank line in the HTML source adversely impacts Github's change view, and does not have any effect on the rendered output. It should be removed.